### PR TITLE
API key added

### PR DESCRIPTION
**Pull request by Meggie-H**
I just added an API key so that I can make more request to the API.

### CODE REVIEW

**Review by yanevdw — APPROVED** [review]
Nicely done!

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,5 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+**environments
```
Comment (tanadendere) [inline .gitignore:26]: Nice 🤫

**Review by tanadendere — APPROVED** [review]
Nice, I know this will help you not be as delayed in your progress while waiting for the hour to pass. Well done on remembering to add the key to git ignore.

File: .gitignore (hunk 1 above)
Comment (Meggie-H) [reply .gitignore:26]: Thank you! Much appreciated

**Pull request merged**